modules/react/layout/lib/StylingBox.tsx

```diff
@@ -0,0 +1,180 @@
+import * as React from 'react';
+import isPropValid from '@emotion/is-prop-valid';
+import styled from '@emotion/styled';
+import {createComponent} from '@workday/canvas-kit-react/common';
+import {createStyles, CSProps, handleCsProp} from '@workday/canvas-kit-styling';
+// eslint-disable-next-line @emotion/no-vanilla
+import {css} from '@emotion/css';
+
+import {boxStyleFn} from './Box';
+
+import {CommonStyleProps} from './utils/styleProps';
+
+/**
+ * StylingBox Props
+ * ---
+ * Common style props + children
+ *
+ * - background
+ * - border
+ * - color
+ * - depth
+ * - flexItem
+ * - gridItem
+ * - layout
+ * - other
+ * - position
+ * - space
+ * - text
+ */
+export type StylingBoxProps = CommonStyleProps &
+  CSProps & {
+    children?: React.ReactNode;
+  };
+
+const omittedProps = [
+  'display',
+  'color',
+  'height',
+  'overflow',
+  'width',
+  'border',
+  'background',
+  'fontSize',
+  'fontWeight',
+  'fontFamily',
+  'letterSpacing',
+  'lineHeight',
+  'textAlign',
+  'opacity',
+  'textDecoration',
+  'textOverflow',
+  'textTransform',
+  'textShadow',
+  'whiteSpace',
+  'wordBreak',
+];
+
+const shouldForwardProp = (prop: string) => {
+  return isPropValid(prop) && !omittedProps.includes(prop);
+};
+
+const stylingBoxStyles = createStyles({
+  boxSizing: 'border-box',
+});
```

Comment:
This is the tidbit you mentioned, using createStyles so that things can be overridden
 yeah?